.github/workflows/create-release.yml

```diff
@@ -1,279 +1,539 @@
 name: Create Release
+# Add a matrix for the node versions to check against
```

Comment:
Is this a TODO? It would be great to reduce duplication, but I won't require it for approval

Reply:
This was just a thought, eventually is not an easy implementation/improvement to automate something that will happen once per year. Removed it https://github.com/paketo-community/ubi-base-stack/pull/7/commits/0c29831235612962949c6ed03d6e7e7d067e52e4